scipy/linalg/tests/test_basic.py

```diff
@@ -1889,14 +1889,6 @@ def test_basic2(self):
         y = solve(circulant(c), b)
         assert_allclose(x, y)
 
```

Comment:
# add this back to check that `solve_circulant`'s reinvention of batch support isn't changed.
```suggestion

    def test_basic3(self):
        # b is a 3-d matrix.
        c = np.array([1, 2, -3, -5])
        b = np.arange(24).reshape(4, 3, 2)
        x = solve_circulant(c, b)
        y = solve(circulant(c), b.reshape(4, -1)).reshape(b.shape)
        assert_allclose(x, y)
```